corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -49,14 +56,73 @@
               .stacked(true)
             ;
             chart.yAxis.tickFormat(d3.format(',.0f'));
-            d3.select('#chart svg')
+            chart.color(["#004ebc", "#e53e30"]);
+            d3.select('#perform_chart svg')
                 .datum([performingSeries, activeNonPerformingSeries])
                 .call(chart);
             nv.utils.windowResize(chart.update);
             return chart;
         });
     }
 
+    function parseToUTCDate(month){
```

Comment:
how does this parse a month to a UTC date? what exactly is being passed in? we use moment.js on a few of our pages which is very helpful for working with dates:http://momentjs.com/